build.gradle

```diff
@@ -9,22 +9,10 @@ buildscript {
 
     def sdkTargetJavaVersion = JavaVersion.VERSION_1_8
 
-    // We are restricted to room version 2.2.6 since we still offer Java 7 support. Upgrading to
-    // newer versions of room that require Java 8 will introduce a breaking change to our SDK
-    def java7SafeRoomVersion = "2.2.6"
-
-    // On M1 Macs, switch room version to 2.4.0 to fix java.lang.ExceptionInInitializerError
-    def m1MacSafeRoomVersion = "2.4.0"
-
-    // The 'CI' build variable will evaluate to 'true' when build script is executed via GitHub Actions
-    // Ref: https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
-    def roomVersion = System.getenv('CI') && !System.getenv('SECURITY_SCAN') ? java7SafeRoomVersion : m1MacSafeRoomVersion
-
     ext.versions = [
             "kotlin"         : "1.7.10",
             "androidxTest"   : "1.4.0",
-            // NEXT MAJOR VERSION: Use a single up-to-date room version once we remove Java 7 support in favor of Java 11
-            "room"           : roomVersion,
+            "room"           : "2.4.0",
```

Comment:
This clean up looks great!

Reply:
The mastermind behind this was @sshropshire!! Thank you!!